dandi/tests/test_files.py

```diff
@@ -236,11 +236,11 @@ def test_find_dandi_files_with_bids(tmp_path: Path) -> None:
             dandiset_path=tmp_path,
             bids_dataset_description_ref=ANY,
         ),
-        BIDSDatasetDescriptionAsset(
+        GenericBIDSAsset(
```

Comment:
so nested BIDS dataset would not be considered a bids dataset? (nothing in PR description hints on the change in behavior for such aspect)

Reply:
True, though a lot of stuff has happened that's not in the PR description. I added that one though, since it is one of the bigger things.